server/command.go

```diff
@@ -327,6 +329,61 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 	return false, nil
 }
 
+func (p *Plugin) runRemoveCommand(args []string, extra *model.CommandArgs) (bool, error) {
+	message := strings.Join(args, " ")
```

Comment:
If we already do this to create the todo, it is ok, but this may create some issues with todos with extra spaces, if I am not mistaken. For example a todo with a message `two  spaces`, when you try to delete, even if you write `two  spaces`, it will get into this variable as `two spaces`.

Reply:
As we are now adopting a different approach, this comment is no longer valid. Therefore, we are resolving this.